internal/pkg/experimental/action/operator_uninstall.go

```diff
@@ -0,0 +1,70 @@
+package action
+
+import (
+	"context"
+	"fmt"
+	"strings"
+	"time"
+
+	olmv1 "github.com/operator-framework/operator-controller/api/v1alpha1"
+	apierrors "k8s.io/apimachinery/pkg/api/errors"
+	"k8s.io/apimachinery/pkg/types"
+	"k8s.io/apimachinery/pkg/util/wait"
+	"sigs.k8s.io/controller-runtime/pkg/client"
+
+	"github.com/operator-framework/kubectl-operator/pkg/action"
+)
+
+type OperatorUninstall struct {
+	config *action.Configuration
+
+	Package string
+
+	Logf func(string, ...interface{})
+}
+
+func NewOperatorUninstall(cfg *action.Configuration) *OperatorUninstall {
+	return &OperatorUninstall{
+		config: cfg,
+		Logf:   func(string, ...interface{}) {},
+	}
+}
+
+func (u *OperatorUninstall) Run(ctx context.Context) error {
+	opKey := types.NamespacedName{Name: u.Package}
+	op := &olmv1.Operator{}
+	op.SetName(opKey.Name)
+	op.SetGroupVersionKind(olmv1.GroupVersion.WithKind("Operator"))
+
+	lowerKind := strings.ToLower(op.GetObjectKind().GroupVersionKind().Kind)
+	if err := u.config.Client.Delete(ctx, op); err != nil && !apierrors.IsNotFound(err) {
+		return fmt.Errorf("delete %s %q: %v", lowerKind, op.GetName(), err)
+	}
+	return waitForDeletion(ctx, u.config.Client, op)
+}
+
+func objectKeyForObject(obj client.Object) types.NamespacedName {
+	return types.NamespacedName{
+		Namespace: obj.GetNamespace(),
+		Name:      obj.GetName(),
+	}
+}
+
+func waitForDeletion(ctx context.Context, cl client.Client, objs ...client.Object) error {
+	for _, obj := range objs {
+		obj := obj
+		lowerKind := strings.ToLower(obj.GetObjectKind().GroupVersionKind().Kind)
+		key := objectKeyForObject(obj)
+		if err := wait.PollImmediateUntil(250*time.Millisecond, func() (bool, error) {
```

Comment:
Could you create a constant for this poll time in the `action` package? Since a couple spots use the same value.